web/src/components/common/FormRow.tsx

```diff
@@ -44,19 +60,57 @@ const Input = styled.input`
   box-shadow: 4px 2px 4px rgba(0, 0, 0, 0.25);
 `;
 
+const ErrorContainer = styled.div`
+  height: 12px;
+  margin-left: 15px;
+  margin-top: 5px;
+
+  font-size: 12px;
+  line-height: 10px;
+  color: var(--bright-red);
+`;
+
+type ReactHookFormErrorMessage =
+  | string
+  | React.ReactElement<
+      any,
+      | string
+      | ((
+          props: any
+        ) => React.ReactElement<
+          any,
+          string | any | (new (props: any) => React.Component<any, any, any>)
+        > | null)
+      | (new (props: any) => React.Component<any, any, any>)
+    >
+  | undefined;
```

Comment:
Oh I took a look and you should actually:
```
import {FieldError, NestDataObject} from 'react-hook-form';
```
```suggestion
type ReactHookFormErrorMessage = string | NestDataObject<any, FieldError> | undefined;
```

Reply:
ohh thanks for looking into this!